packages/core-js/utils/getEvents.js

```diff
@@ -0,0 +1,26 @@
+async function getPastEvents({ contract, eventName, provider, fromBlock, toBlock }) {
```

Comment:
Is the interactive task using this? If not, I'd consider removing the file for now. We can always go back to the original CLI implementation and grab it.

Reply:
I'll check and remove if not

Reply:
Don't needed. Removed.